client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -0,0 +1,132 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { omitBy, isUndefined } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { RESULT_ARTICLE, RESULT_TOUR, RESULT_VIDEO } from './constants';
+import Button from 'components/button';
+import Dialog from 'components/dialog';
+import ResizableIframe from 'components/resizable-iframe';
+import { decodeEntities, preventWidows } from 'lib/formatting';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { getSearchQuery } from 'state/inline-help/selectors';
+import { requestGuidedTour } from 'state/ui/guided-tours/actions';
+
+class InlineHelpRichResult extends Component {
+	static propTypes = {
+		result: PropTypes.object,
+	};
+
+	state = {
+		showDialog: false,
+	};
+
+	handleClick = event => {
+		event.preventDefault();
+		const { href } = event.target;
+		const { type } = this.props;
+		const { tour } = this.props.result;
+		const tracksData = omitBy(
```

Comment:
Should this include a `siteId` / `blog_id` when applicable? I'm not really convinced it should, but thought I'd ask.